### ISSUE TITLE
CI: crucible-llvm-tests fail on macOS

### ISSUE THREAD

**Issue by RyanGlScott** [GaloisInc/crucible#741]
From a [recent macOS CI job](https://github.com/GaloisInc/crucible/runs/2571681074#step:16:149):

```
Test suite crucible-llvm-tests: RUNNING...
Tests
<elided>
    global-struct.c
      compile global-struct:                         OK (0.03s)
      parse global-struct bitcode:                   Failed to parse test/c/global-struct.bc
Couldn't parse LLVM bitcode from filetest/c/global-struct.bc
Error {errContext = ["METADATA_COMPOSITE_TYPE","METADATA_BLOCK","METADATA_BLOCK_ID","value symbol table","MODULE_BLOCK","Bitstream"], errMessage = "Invalid record size: 21\nExpected size between 16 and 18\nAre you sure you're using a supported version of LLVM/Clang?\nCheck here: https://github.com/GaloisInc/llvm-pretty-bc-parser\n"}
FAIL
        test/Tests.hs:219:
        expected: ""
         but got: "Couldn't parse LLVM bitcode from filetest/c/global-struct.bc\nError {errContext = [\"METADATA_COMPOSITE_TYPE\",\"METADATA_BLOCK\",\"METADATA_BLOCK_ID\",\"value symbol table\",\"MODULE_BLOCK\",\"Bitstream\"], errMessage = \"Invalid record size: 21\\nExpected size between 16 and 18\\nAre you sure you're using a supported version of LLVM/Clang?\\nCheck here: https://github.com/GaloisInc/llvm-pretty-bc-parser\\n\"}"
<elided>

3 out of 34 tests failed (3.58s)

Test suite crucible-llvm-tests: FAIL
```

**Comment by robdockins** [comment]
CF #577

**Comment by RyanGlScott** [comment]
I believe this was fixed in #753.

**Comment by kquick** [comment]
I believe, you believe, we all relieve together.
